package.json

```diff
@@ -6,9 +6,11 @@
         "node": ">=8"
     },
     "scripts": {
-        "start": "react-scripts start",
+        "start": "npm-run-all -p watch:css start:app build",
+        "start:app": "react-scripts start",
         "build": "npm-run-all build:*",
         "build:app": "react-scripts build",
+        "build:css": "stylus --sourcemap ./src/components/views/styl/* --out ./src/components/views/css/",
```

Comment:
Correct me if I'm mistaken, but I think that we need to do this to ensure that everything is built correctly:

```javascript
"build": "npm-run-all --parallel build:css build:app",
"build:css": "stylus --sourcemap ./src/components/views/styl/* --out ./src/components/views/css/",
"build:app": "react-scripts build",
```

That approach would force the Stylus to be compiled before `react-scripts` bundles the CSS. As it turns out, we may need to make this fix on Test Tube, too.

Reply:
Yeah probably a good call to be more explicit.

Reply:
Ah, my mistake. I mean sequential. And it turns out sequential is by default, but I think calling out `build:css` and `build:app` explicitly in that order helps.

Reply:
Oh you did that. Nice!